### DISCUSSION TITLE
How to configure gfm options

### DISCUSSION THREAD

**Discussion by HJ-fukura** [Milkdown/milkdown#1062] [General]
I can configure options for remark-stringify with `remarkStringifyOptionsCtx`.

So how can I set options for remark-gfm like `singleTilde` ?

https://github.com/remarkjs/remark-gfm

**Comment by Saul-Mirone** [comment]
Currently there's no way to configure it, I'll add the support soon.
https://github.com/Milkdown/milkdown/issues/1064
